commit 916cea9f4810e780fbf8e804d82ce35d84e4506e
Author: Matthias Reichl <hias@horus.com>
Date:   Fri Feb 2 20:30:42 2018 +0100

    ASoC: iqaudio-dac: fix S24_LE format
    
    Remove set_bclk_ratio call so 24-bit data is transmitted in
    24 bclk cycles.
    
    Signed-off-by: Matthias Reichl <hias@horus.com>

diff --git a/sound/soc/bcm/iqaudio-dac.c b/sound/soc/bcm/iqaudio-dac.c
index 1ee4097c8463..4583d199fd83 100644
--- a/sound/soc/bcm/iqaudio-dac.c
+++ b/sound/soc/bcm/iqaudio-dac.c
@@ -43,18 +43,6 @@ static int snd_rpi_iqaudio_dac_init(struct snd_soc_pcm_runtime *rtd)
 	return 0;
 }
 
-static int snd_rpi_iqaudio_dac_hw_params(struct snd_pcm_substream *substream,
-	struct snd_pcm_hw_params *params)
-{
-	struct snd_soc_pcm_runtime *rtd = substream->private_data;
-	struct snd_soc_dai *cpu_dai = rtd->cpu_dai;
-
-	unsigned int sample_bits =
-		snd_pcm_format_physical_width(params_format(params));
-
-	return snd_soc_dai_set_bclk_ratio(cpu_dai, sample_bits * 2);
-}
-
 static void snd_rpi_iqaudio_gpio_mute(struct snd_soc_card *card)
 {
 	if (mute_gpio) {
@@ -109,11 +97,6 @@ static int snd_rpi_iqaudio_set_bias_level(struct snd_soc_card *card,
 	return 0;
 }
 
-/* machine stream operations */
-static struct snd_soc_ops snd_rpi_iqaudio_dac_ops = {
-	.hw_params = snd_rpi_iqaudio_dac_hw_params,
-};
-
 static struct snd_soc_dai_link snd_rpi_iqaudio_dac_dai[] = {
 {
 	.cpu_dai_name	= "bcm2708-i2s.0",
@@ -122,7 +105,6 @@ static struct snd_soc_dai_link snd_rpi_iqaudio_dac_dai[] = {
 	.codec_name	= "pcm512x.1-004c",
 	.dai_fmt	= SND_SOC_DAIFMT_I2S | SND_SOC_DAIFMT_NB_NF |
 				SND_SOC_DAIFMT_CBS_CFS,
-	.ops		= &snd_rpi_iqaudio_dac_ops,
 	.init		= snd_rpi_iqaudio_dac_init,
 },
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1193_916cea9f4810e780fbf8e804d82ce35d84e4506e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1193_916cea9f4810e780fbf8e804d82ce35d84e4506e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
